Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -45,9 +45,20 @@
 - type: damageModifierSet
   id: Glass
   coefficients:
-    Blunt: 0.5
+    Blunt: 1
+    Slash: 0.5
+    Piercing: 1.1
+    Heat: 0
+    Shock: 0
+  flatReductions:
+    Blunt: 5
+    
+- type: damageModifierSet
+  id: RGlass
+  coefficients:
+    Blunt: 0.8
     Slash: 0.5
-    Piercing: 1.5
+    Piercing: 0.7
     Heat: 0
     Shock: 0
   flatReductions:
```

Comment:
imo:
- remove the blunt flatreduction
- add a flat reduction for piercing
- reduce piercing to like 0.8 or something, buff blunt to 1.2 (at least for normal glass)

Reply:
Remember that w/o the flat reduction, people will be able to punch through windows

Reply:
ehh true this is fine then